content/post/daily-bugle-boot2-root-writeup copy.md

```diff
@@ -0,0 +1,734 @@
+---
+title: "Daily Bugle Boot2Root Write-Up"
+date: 2020-11-27T00:00:00-04:00
+author: "Nate Catelli"
+tags: ["ctf", "boot2root", "hacking", "writeup", "tryhackme"]
+description: "A boot2root writeup of the Daily Bugle host from TryHackMe"
+type: "post"
+draft: false
+---
+
+## Introduction:
+The Daily Bugle challenge was exceptionally difficult compared to a few of the other hosts I've attacked in the past. Not specifically for any technical reason but it provided many opportunities to rabbithole if I wasn't dilligent in both thinking about how I looked for information but also in what I documented. I will attempt to call out the points that I really rabbitholed on but I think it will be very difficult to express in written form.
+
+## Environment
+The attack takes place on a flat network consisting of the attack host, a freshly-booted Kali Linux livecd, and the the target host. I knew very little about the host prior to the attack other than that there would be two flags on the host, a user and root flag.
```

Comment:
I was given hardly any information about the host prior to the attack other than...